src/m365/teams/commands/team/team-add.ts

```diff
@@ -89,31 +89,32 @@ class TeamsTeamAddCommand extends GraphCommand {
         };
         requestBody.description = args.options.description;
       }
-
     }
     else {
       requestBody = {
-        'template@odata.bind': `https://graph.microsoft.com/beta/teamsTemplates('standard')`,
+        'template@odata.bind': `https://graph.microsoft.com/v1.0/teamsTemplates('standard')`,
         displayName: args.options.name,
         description: args.options.description
       }
     }
 
     const requestOptions: AxiosRequestConfig = {
-      url: `${this.resource}/beta/teams`,
+      url: `${this.resource}/v1.0/teams`,
       headers: {
-        accept: 'application/json;odata.metadata=none',
-        'content-type': 'application/json;odata.metadata=none'
+        'accept': 'application/json;odata=nometadata'
       },
       data: requestBody,
-      responseType: 'json'
+      responseType: 'stream'
     };
 
     request
       .post(requestOptions)
       .then((res: any): Promise<TeamsAsyncOperation> => {
+        // Avoid Error: Converting circular structure to JSON
+        JSON.stringify(res.headers);
```

Comment:
What's the purpose of this line as its outcome is not being used anywhere?

Reply:
Hi @waldekmastykarz 
Previously, I came across an error `Converting circular structure to JSON` and JSON.stringify resolved it (although not used anywhere else). Reference from: https://github.com/axios/axios/issues/836

Now that, when I changed the code to OData 4 values (odata.metadata=none) this error seems to be resolved. Removed this code from recent PR.
Thank you.

Reply:
Thanks for double checking. I think I might've stumbled upon the same issue when building the file convert pdf command and fixed it in the request wrapper. Nevertheless, good to hear the issue is gone with the correct accept header 👍